robottelo/cli/base.py

```diff
@@ -249,7 +244,7 @@ def info(cls, options=None):
                 .format(cls.__name__)
             )
 
-        result = cls.execute(cls._construct_command(options), expect_csv=False)
```

Comment:
Dropped all `expect_csv=False` which indicates that this commands does not need to parse the output.